src/main/java/ip/Parser.java

```diff
@@ -33,6 +27,11 @@ public Command parse(String input) {
             switch (command) {
                 case "bye": return new ExitCommand();
                 case "list": return new ListCommand();
+                case "find":
+                    if (!inputSc.hasNext()){
+                        throw new Duke.DukeException("The keyword to search is missing :p");
+                    }
+                    return new FindCommand(inputSc.nextLine().trim());
                 case "done":
```

Comment:
You could possible add a styling guide to your intelliJ which could alleviate all these styling issues. Do ensure you follow the checkstyle given for switch and case.